recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/eth0.network

```diff
@@ -1,5 +1,6 @@
 [Match]
 Name=eth0
+KernelCommandLine=!nfsroot
```

Comment:
I guess we do not need it to be configured in user space if we have dhcp in kernel command line either.

Reply:
and what if not?

Reply:
Well, I did mean addition of another condition like !dhcp. But we would like to have a userspace dhcp client running even if kernel side dhcp client timed out without address received.